tests/plugins/balloonpanel/positioningnonstatic.js

```diff
@@ -0,0 +1,104 @@
+/* bender-tags: editor,balloonpanel */
+/* bender-ckeditor-plugins: balloonpanel,toolbar,wysiwygarea,basicstyles */
+/* bender-include: ./_helpers/tools.js */
+/* global balloonTestsTools */
+
+( function() {
+	'use strict';
+
+	var stubs = [];
+
+	bender.editors = {
+		classic: {
+			name: 'editor1',
+			startupData: '<p><strong>Hello</strong> world</p>',
+			config: {
+				width: 300,
+				height: 300,
+				allowedContent: true
+			}
+		},
+		divarea: {
+			name: 'editor2',
+			startupData: '<p><strong>Hello</strong> world</p>',
+			config: {
+				width: 300,
+				height: 300,
+				allowedContent: true,
+				extraPlugins: 'divarea'
+			}
+		},
+		inline: {
+			name: 'editor3',
+			startupData: '<p><strong>Hello</strong> world</p>',
+			creator: 'inline',
+			config: {
+				width: 300,
+				height: 300,
+				allowedContent: true
+			}
+		}
+
+	};
+
+	function calculatePositions( editor, panelWidth, panelHeight ) {
+		var panel = new CKEDITOR.ui.balloonPanel( editor, {
+			title: 'Test panel #1',
+			width: panelWidth,
+			height: panelHeight
+		} );
+		var element = editor.editable().findOne( 'strong' );
+
+		panel.attach( element );
+
+		return {
+			elementRect: panel._getAbsoluteRect( element ),
+			triangleTip: balloonTestsTools.getTriangleTipPosition( panel ),
+			panel: panel
+		};
+	}
+
+	var tests = {
+		setUp: function() {
```

Comment:
`setUp` and `tearDown` are run before/after each test. While this is the only place where you add a stub to `stubs` array it looks like there is always only one stub, so it would be easier just to store it globally without `stubs` array. You may rename it to `getViewPaneSizeStub` (or something similar) for clarity.